### ISSUE TITLE
<button type="submit"> becomes an <input>

### ISSUE THREAD

**Issue by f0rmat1k** [kizu/bemto#63]
There is code in tool:
```jade
  if attributes.type
    - newTag = 'input';
```

There are situation, when button must have `type=submit` attribute. But i getting `<input>`, which breaks me view.

**Comment by f0rmat1k** [comment]
This
```jade
else if attributes.src
    - newTag = 'img';
```
also looks very strange, because many of tags has `src` attribute, not only `img`. E.g.
`+b('audio')(src="http://yandex.ru")` became an image.

In my opinion all section `//- Attributes context checks` is evil.

**Comment by kizu** [comment]
That was fixed in a 1.0.0, now the context matters only for defaulted tags, while hardcoded ones won't change based on attributes.
